src/test/java/plugins/GradlePluginTest.java

```diff
@@ -23,32 +23,37 @@
  */
 package plugins;
 
+import org.jenkinsci.test.acceptance.Matchers;
 import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
 import org.jenkinsci.test.acceptance.junit.WithPlugins;
 import org.jenkinsci.test.acceptance.plugins.gradle.GradleInstallation;
 import org.jenkinsci.test.acceptance.plugins.gradle.GradleStep;
+import org.jenkinsci.test.acceptance.po.Build;
 import org.jenkinsci.test.acceptance.po.FreeStyleJob;
 import org.junit.Test;
 
+import java.util.regex.Pattern;
+
 @WithPlugins("gradle")
 public class GradlePluginTest extends AbstractJUnitTest {
 
     @Test
     public void run_gradle_scirpt() {
-        GradleInstallation.installGradle(jenkins, "gradle-1.5", "1.5");
+        final String gradleInstallationName = "Default";
+        GradleInstallation.installGradle(jenkins, gradleInstallationName, GradleInstallation.LATEST_VERSION);
 
-        FreeStyleJob job = jenkins.jobs.create();
+        final FreeStyleJob job = jenkins.jobs.create();
         job.copyResource(resource("/gradle_plugin/script.gradle"), "build.gradle");
-        GradleStep step = job.addBuildStep(GradleStep.class);
-        step.useVersion("gradle-1.5");
+        final GradleStep step = job.addBuildStep(GradleStep.class);
+        step.useVersion(gradleInstallationName);
         step.tasks.set("hello");
         step.switches.set("--quiet");
         job.save();
 
-        job.startBuild().shouldSucceed()
-                .shouldContainsConsoleOutput("Hello world!")
-                .shouldContainsConsoleOutput("gradle.* --quiet")
-        ;
+        final Build build = job.startBuild();
+        build.shouldSucceed();
+        Matchers.containsRegexp("Hello world!", Pattern.MULTILINE);
```

Comment:
This does not check anything! The JavaDoc of the deprecated method is misleading, we should change that too.

The correct assertion is 

`assertThat(build.getConsole(), containsString("Hello World!"));`

(or containsRegexp)